wdl/GatherSampleEvidence.wdl

```diff
@@ -319,6 +339,9 @@ workflow GatherSampleEvidence {
     Int? melt_read_length = MELT.read_length_out
     Float? melt_insert_size = MELT.insert_size_out
 
+    File? scramble_vcf = Scramble.insertions_vcf
+    File? scramble_index = Scramble.insertions_index
```

Comment:
Unless I'm missing something, it doesn't look like the deletions vcf was ever hooked up here? Does this mean your batch run with deletions enabled wasn't correct?

Reply:
Damn.  Yeah, it probably does.

Reply:
Ok, let's not bother with running that again. The runtimes are too high and variable with deletion calling.